tapchannel/aux_invoice_manager.go

```diff
@@ -247,6 +248,34 @@ func (s *AuxInvoiceManager) handleInvoiceAccept(_ context.Context,
 	return resp, nil
 }
 
+// assetIDFromQuote retrieves the quote by looking up the rfq manager's maps of
+// accepted quotes based on the passed rfq ID. If there's a match, the asset
+// ID is returned.
+func (s *AuxInvoiceManager) assetIDFromQuote(
```

Comment:
To avoid us needing to touch this again once we add group key support, should we turn this into `identifierFromQuote()` instead and return the identifier directly?